<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HotkeyItemWidgetClass</class>
 <widget class="QWidget" name="HotkeyItemWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>49</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>49</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>49</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HotkeyItemWidget</string>
  </property>
  <widget class="QKeySequenceEdit" name="seqDec">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClearInc">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClearDec">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QKeySequenceEdit" name="seqInc">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Player 0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>131</width>
     <height>6</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>131</width>
     <height>6</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../RC/Versus.qrc"/>
 </resources>
 <connections/>
</ui>
